dbt/adapters/spark/connections.py

```diff
@@ -162,12 +162,17 @@ def open(cls, connection):
                 break
             except Exception as e:
                 exc = e
-                if hasattr(e, 'message') and 'pending' in (e.message.lower()):
-                    logger.warning("Warning: {}\n\tRetrying in {} seconds "
-                                   "({} of {})".format(e.message,
-                                                       connect_timeout,
-                                                       i + 1,
-                                                       connect_retries))
+                if not hasattr(e, 'message') or if e.message is None:
```

Comment:
@drewbanin Seeing an error in this line:
```
dbt debug
dbt version: 0.13.0
python version: 3.7.2
python path: /usr/local/opt/python/bin/python3.7
os info: Darwin-18.5.0-x86_64-i386-64bit
Using profiles.yml file at /Users/jerco/.dbt/profiles.yml

Encountered an error:
invalid syntax (connections.py, line 165)
```
I realize I have a working install at commit 4c92eb0

Reply:
That extra `if` is the problem. Though really this could just be `if getattr(e, 'message', None) is None:`